static/test/javascripts-legacy/spec/common/ui/relativedates.spec.js

```diff
@@ -1,192 +0,0 @@
-                },
-                'moreThanTwoDaysAgo': {
-                    'date'           :      '2012-08-09T08:34:00+01:00',
-                    'expectedOutput' :      'Thursday 9 Aug 2012',
-                    'expectedShortOutput' : '4d',
-                    'expectedMedOutput' :   '4d ago',
-                    'expectedLongOutput' :  'Thursday 9 Aug 2012'
-                },
-                'moreThanFiveDaysAgo': {
-                    'date'           :      '2012-08-05T21:30:00+01:00',
-                    'expectedOutput' :      '5 Aug 2012',
-                    'expectedShortOutput' : '5 Aug 2012',
-                    'expectedMedOutput'   : '5 Aug 2012',
-                    'expectedLongOutput' :  '5 Aug 2012'
-                },
-                'oneMinuteAgoInAnotherTimeZone': {
-                    'date'           :      '2012-08-13T12:58:40+02:00',
-                    'expectedOutput' :      '1m',
-                    'expectedShortOutput' : '1m',
-                    'expectedMedOutput'   : '1m ago',
-                    'expectedLongOutput' :  '1 minute ago'
-                }
-            };
-
-        for (var category in datesToTest) {
-            /*eslint-disable no-loop-func*/
-            describe('Show relative dates for timestamps formatted as YYYY-MM-DD HH:MM:SS [' + category + ']', function () {
-                    var d     = datesToTest[category],
-                        epoch = Date.parse(d.date);
-                    it('standard', function () {
-                        expect(RelativeDates.makeRelativeDate(epoch)).toBe(d.expectedOutput);
-                    });
-                    it('short', function () {
-                        // Do the same but in short format
-                        expect(RelativeDates.makeRelativeDate(epoch, { format: 'short' })).toBe(d.expectedShortOutput);
-                    });
-                    it('med', function () {
-                        expect(RelativeDates.makeRelativeDate(epoch, { format: 'med' })).toBe(d.expectedMedOutput);
-                    });
-                    it('long', function () {
-                        // and long format
-                        expect(RelativeDates.makeRelativeDate(epoch, { format: 'long' })).toBe(d.expectedLongOutput);
-                    });
-                });
-            /*eslint-enable no-loop-func*/
-        }
-
-        it('Return the input date if said date is in the future', function () {
-            expect(RelativeDates.makeRelativeDate(Date.parse(epochBug))).toBeFalsy();
-        });
-
-        it('Fail politely if given non-date / invalid input for either argument', function () {
-            expect(RelativeDates.makeRelativeDate('foo')).toBeFalsy();
-        });
-
-        it('Fail politely if the date is older than a \'notAfter\' value', function () {
-            expect(RelativeDates.makeRelativeDate(Date.parse(fakeNow), {notAfter: 3600})).toBeFalsy();
```

Comment:
Turns out Date.parse on a number returns `NaN` so wasn't really testing what it was supposed to.

Reply:
I.e. as far as I can tell it was only passing by chance.